label_studio/tests/data_import/test_models.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from django.core.files.base import ContentFile
+
+from data_import.models import FileUpload
+
+pytestmark = pytest.mark.django_db
+
+
+class TestFileUpload:
+    @pytest.fixture
+    def jsonl_file(self, configured_project):
+        return FileUpload.objects.create(
```

Comment:
I think it's better to use API for creating objects. 

Because in this case you can refactor implementation as much as you want, and you don't need to touch test on every change.

Reply:
@triklozoid  Sorry, I didn't understand. Are you suggesting to make an API request to create the FileUpload ?

Reply:
@guilhermemachado26 

Yes, I suggest to make test on API level. 
* call import API with jsonl body
* call tasks list API to check, that import works well. 

Example of test: https://github.com/heartexlabs/label-studio/blob/fb-dev-239-api/label_studio/tests/data_manager/test_views_api.py#L10

Reply:
@guilhermemachado26  Why do you think we need a separate test for the read_tasks_list_from_jsonl method? 
For example, if I will add the required field for the FileUpload model, this test will fail, because it uses `FileUpload.objects.create`. But this change can be completely unrelated to JsonL logic. So it will increase support costs for the project. It's ok if we can catch some bugs but what bugs we can catch, that is not covered with API level test?

Reply:
@triklozoid I do agree about stop using `FileUpload.objects.create`. I'd like to suggest [FactoryBoy](https://factoryboy.readthedocs.io/en/stable/orms.html) to create these objects, in way that we would only need to add changes in one single place.

I believe its a good practice each test only test one function. Testing methods individually keeps the code more organized and simple, once API tests have extra overhead. Also, the method might be called from other parts of the project, not only from the API flow. If we decide to update/remove the API, we will lose test coverage.

Reply:
Just to be clear, I'm completely agree to test the API to check if the whole flow works as expected and the method was called, but some methods may have complex implementation, different inputs and couple of edge cases to be tested, and maybe they should be tested individually.

Reply:
> but some methods may have complex implementation, different inputs and couple of edge cases to be tested, and maybe they should be tested individually.

Generally, I agree, but for this particular function and tests, I see neither complex implementation or edge cases. 

If we already have it, okay, let's keep it. But for similar functions in the future, I think we don't need tests.

Reply:
Sure, thanks for your feedback